src/main/java/org/jenkinsci/plugins/mesos/JenkinsScheduler.java

```diff
@@ -551,6 +551,20 @@ void setDriver(SchedulerDriver driver) {
   }
 
   @VisibleForTesting
+  String findRoleForPorts(Offer offer) {
+
+      String role = null;
```

Comment:
We could set the initial value to "*" which is what the old functionality was anyway? That would mitigate the concern with the use of the return value where this is used on line 771?